### DISCUSSION TITLE
videos with .ts extension

### DISCUSSION THREAD

**Discussion by parryhotter007** [streamlink/streamlink#5726] [Q&A] [answered]
### Description

I have a provider who has channels in mpegts format (.ts). There's no plugin in streamlink to handle it. Can streamlink not handle it?

**Comment by bastimeyer** [comment, marked as answer]
https://streamlink.github.io/cli/protocols.html

`httpstream://https://host/file.ts`

**Comment by parryhotter007** [comment]
Thanks mate.
